server/asset/eth/config.go

```diff
@@ -24,3 +48,65 @@ type configuredTokenGases struct {
 	Swap   uint64 `ini:"swap"`
 	Redeem uint64 `ini:"redeem"`
 }
+
+// loadConfig loads the config from file. If no values are found for
+// RPCListen or RPCCert in the specified file, default values will be used. If
+// configPath is an empty string, loadConfig will attempt to read settings
+// directly from the default geth.conf file path. If there is no error, the
+// module-level chainParams variable will be set appropriately for the network.
+func loadConfig(configPath string, net dex.Network, logger dex.Logger) (*config, error) {
+	switch net {
+	case dex.Simnet:
+	case dex.Testnet:
+	case dex.Mainnet:
+		// TODO: Allow. When?
+		return nil, fmt.Errorf("eth cannot be used on mainnet")
+	default:
+		return nil, fmt.Errorf("unknown network ID: %d", net)
+	}
+
+	cfg := new(config)
+
+	// NOTE: ipc only is deprecated. Remove this at some point.
+	if strings.HasSuffix(configPath, ".ipc") || configPath == "" {
+		ipc := configPath
+		if ipc == "" {
+			ipc = defaultIPC
+		}
+		cfg.ADDR = ipc
+		logger.Warnf("Geth ipc is set in markets.json. This should be "+
+			"moved to a new file somewhere and the file location included "+
+			"in the markets.json.%s", exConfStr)
```

Comment:
I mean the ipc location, like "/home/user/geth.ipc" should be stuck in a 'conf' file. We want the config file now, that may have an ipc location in it, rather than jsut the ipc file location.

Reply:
Ah, ok. "This" I thought referred to a file itself being moved, you mean the string that specifies the path should be moved *into* a new config file and the markets.json point to that file.  Will you try to rephrase this warning a bit?

Reply:
Yes.

Reply:
Attempted to make this more clear... I just hope to use a config file like the other assets, as we need more info now and there may be more things we need to set there in the future. This shouldn't be a change for anyone but us.